c/busybox-1.22.0/busybox_sv_comp-mmap.h

```diff
@@ -0,0 +1,24 @@
+void *mmap(void *addr, size_t length, int prot, int flags, int fd, off_t offset)
+{
+  (void)addr;
+  (void)prot;
+  (void)offset;
+
+  if(flags & MAP_ANONYMOUS)
+  {
+    if(fd != -1)
+      __VERIFIER_error();
+
+    return calloc(length, 1);
```

Comment:
I think here may be a problem. Since only malloc is required to return non-NULL according to the rules, calloc may return NULL on failure. However, mmap's return value on failure is ((void *) -1) (which should be probably also modeled), so if the allocation by calloc fails, mmap returns NULL, which is a valid value for mmap, but the memory was not actually allocated.